maven/bnd-testing-maven-plugin/src/main/java/aQute/bnd/maven/testing/plugin/TestingMojo.java

```diff
@@ -131,8 +126,10 @@ private void testing(Run run) throws Exception {
 			if (run.getErrors().isEmpty())
 				return;
 
-			System.out.println(Strings.join("\n", run.getErrors()));
-			throw new MojoExecutionException("Test had errors");
+			throw new MojoExecutionException(
+					Strings.join("\n", new Object[] {
```

Comment:
yup, as I said earlier, the varargs join doesn't work when the delimiter is the same type as the args you mean to delimit.